### ISSUE TITLE
Create a volunteer welcoming doc

### ISSUE THREAD

**Issue by Lemoert** [Trustroots/community#70]
Containing info such as how TR works, where how-to's can be found, what volunteering entails, good practices (not expecting others to do work unless they've said so themselves), etc. etc.

**Comment by loberto** [comment]
what about https://codi.kanthaus.online/ahoyparty ?
I edited some to make it something like this

**Comment by Lemoert** [comment]
Yup, that looks like a brilliant place to start! 

I have some vague ideas of things I would like to add, but would need to take time to make them a bit more concrete first. Something about expectations and the ebb and flow in volunteering activity (at first it can seem as if an active exclusion is going on when things drop from very much activity to very little without much of an explanation.

**Comment by loberto** [comment]
maybe we want 2 documents.
1 for the basic of work
2 for the ups and down, and turns, of TR volunteer community

**Comment by loberto** [comment]
I wouldn't want to have only one, but never ending.... and I feel that things about the expectation could be long... if it's short, then I take back my previous comment.

**Comment by Lemoert** [comment]
Makes sense, one for the "practical" part and one for the "philosophical" part. Could do them separately and decide based on length. Not much point in keeping a separate doc with two lines in it ;)
